svthalia/concrexit
# Add short description field to events

Closes #2067 



### Summary
Added the feature to add a short description of events for promotion and the newsbrief
### How to test
Steps to test the changes you made:
1. Go to 'site administration'
2. Go to add event
3. Scroll down to 'detail(Show)'
4. There should be a field to enter a caption
